tests/Tinkoff.Trading.OpenApi.Tests/SerilalizationTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+using FluentAssertions;
+
+using Newtonsoft.Json;
+
+using Tinkoff.Trading.OpenApi.Models;
+using Tinkoff.Trading.OpenApi.Tests.TestHelpers;
+
+using Xunit;
+
+namespace Tinkoff.Trading.OpenApi.Tests
+{
+    public class SerilalizationTests
```

Comment:
Лучше озаглавить более приближенно к объекту тестирования
```suggestion
    public class StreamingResponseTests
```